python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -365,22 +365,24 @@ def __get__(self, obj, cls):
             return cls._schema[name]
 
         if self._getter_name:
-            return self._bind_method(obj, self._getter_name)()
+            returnable = self._bind_method(obj, self._getter_name)()
+            if isinstance(returnable, GangaObject):
+                returnable._setParent(self)
+            return returnable
```

Comment:
In this case shouldn't it be ```_setParent(obj)``` rather than not setting the parent if the object is a GangaObject, otherwise it could (does) have the parent of ```None```?
I'm curious how you spotted this breaking anything?